source/Octopus.Versioning/Docker/DockerTag.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+
+namespace Octopus.Versioning.Docker
+{
+    public class DockerTag : IVersion
```

Comment:
Could we also introduce a new `IDockerVersion` interface with just the good bits? Might make handling these elsewhere a bit nicer to reason about.

Reply:
I'll make figuring out the abstraction my task for the next sharpening.